test/e2e/storage/vsphere/persistent_volumes-vsphere.go

```diff
@@ -178,7 +178,7 @@ var _ = utils.SIGDescribe("PersistentVolumes:vsphere", func() {
 		3. Verify that written file is accessible after kubelet restart
 	*/
 	It("should test that a file written to the vspehre volume mount before kubelet restart can be read after restart [Disruptive]", func() {
-		utils.TestKubeletRestartsAndRestoresMount(c, f, clientPod, pvc, pv)
+		utils.TestKubeletRestartsAndRestoresMount(c, f, clientPod)
```

Comment:
A little confused here, if we're removing the pv, pvc from the spec, why does this PR not have any added code which creates PV and PVC somewhere else? Or was it unused to begin with

Reply:
Yeah, PVC/PV dependencies were removed in a different PR, but they didn't remove the variables from the arguments.

Reply:
ack